include/cocaine/rpc/slot.hpp

```diff
@@ -29,6 +29,10 @@
 #include <boost/mpl/transform.hpp>
 #include <boost/optional/optional_fwd.hpp>
 
+namespace cocaine { namespace hpack {
+class header_t;
```

Comment:
AFAIK, we define it in cocaine/forwards.hpp, if not - maybe better to add it there